src/client/Microsoft.Identity.Client/Internal/Requests/Interactive/InteractiveRequest.cs

```diff
@@ -109,7 +109,14 @@ private async Task<MsalTokenResponse> GetTokenResponseAsync(CancellationToken ca
                     return brokerTokenResponse;
                 }
 
+                if (string.Equals(_requestParams.AuthenticationScheme.AccessTokenType, Constants.PoPTokenType))
```

Comment:
Preventing pop broker flow from falling back to the browser when broker is not available.

Reply:
Pls create a work item about the devex around POP.